workspaces/src/types/sdk.rs

```diff
@@ -0,0 +1,18 @@
+use std::convert::TryFrom;
+
+use crate::error::{Error, ErrorKind};
+
+use super::PublicKey;
+
+impl TryFrom<near_sdk::PublicKey> for PublicKey {
```

Comment:
PS: what do you think about providing an inverse conversion?

Reply:
what would the use case be for the inverse? I don't think anything would consume sdk related types besides sending it over the wire through serialization

Reply:
That makes sense. Didn't have any specific use case in mind. Just thought it made sense to provide all the reverse conversions where possible. But if there's no clear use case for this, it's fine to leave out.